File: Assignment1/last2ndEven.java
import java.util.*;

public class last2ndEven
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter number: ");
		int n = sc.nextInt();
		while ((n%10)%2==1)
		{
			n /= 10;
			continue;
		}
		n /= 10;
		while((n%10)%2==1)
		{
			n /= 10;
		}
		System.out.println(n%10);

		sc.close();
	}
}

File: Assignment1/last2ndEven.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter number: ";
    cin >> n;
    while ((n%10)%2==1)
    {
        n /= 10;
        continue;
    }
    n /= 10;
    while(n>0)
    {
        if ((n%10)%2==0)
        {
            cout << n%10 << endl;
            break;
        }
        n /= 10;
    }
    return 0;
}